Changes Line 33 and line 81 (#7057)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/manage/centralized-deployment-of-add-ins.md

Centralized Deployment provides the following benefits:

> - An admin can deploy and assign an add-in directly to a user, to multiple users via a group, or to everyone in the organization (see Admin requirement section for info)

Reviewer: ```suggestion
- An admin can deploy and assign an add-in directly to a user, to multiple users via a group, or to everyone in the organization (see Admin requirement section for information).
```

Check with your organization's Exchange admin to find out which configuration is in use. OAuth connectivity per user can be verified by using the [Test-OAuthConnectivity](/powershell/module/exchange/test-oauthconnectivity) PowerShell cmdlet.

### Admin requirements

In order to deploy an add-in via Centralized Deployement, you need to be either a Global admin or an Exchange admin in the organization.

> [!NOTE]

> >Exchange admin can deploy add-in only if **App Registrations** property is set to true in Azure Active Directory admin center as shown below:

Reviewer: ```suggestion
> An Exchange admin can deploy an add-in only if the **App Registrations** property is set to true in Azure Active Directory admin center as shown in the following image:
```

>

> >![image](https://user-images.githubusercontent.com/89943918/144516704-8874a10d-b540-41f3-ae9d-c07a8d7e143f.png)

Reviewer: ```suggestion
> ![image](https://user-images.githubusercontent.com/89943918/144516704-8874a10d-b540-41f3-ae9d-c07a8d7e143f.png)
```
